src/DotNetOpenAuth.OpenId.RelyingParty/OpenId/RelyingParty/AuthenticationRequest.cs

```diff
@@ -509,8 +509,8 @@ internal class AuthenticationRequest : IAuthenticationRequest {
 				}
 			}
 
-			// Sort endpoints so that the first one in the list is the most preferred one.
-			filteredEndpoints.OrderBy(ep => ep, relyingParty.EndpointOrder);
+            // Sort endpoints so that the first one in the list is the most preferred one.
+            filteredEndpoints = filteredEndpoints.OrderBy(ep => ep, relyingParty.EndpointOrder).ToList();
```

Comment:
The change looks good. But can you fix the whitespace to match the surrounding code (use tabs instead of spaces)?